src/main/java/com/takipi/oss/storage/fs/concurrent/ConcurrentTaskExecutor.java

```diff
@@ -16,10 +16,12 @@
 	private final AtomicInteger threadCount = new AtomicInteger();
 	
 	public ConcurrentTaskExecutor(int maxThreads) {
+	    
+	    int maxAllowedThreads = 500;
 		
-		if (maxThreads > 50) {
-			logger.warn("ConcurrentTaskExecutor cannot have more than 50 threads");
-			maxThreads = 50;
+		if (maxThreads > maxAllowedThreads) {
```

Comment:
why do we keep any limit at all?

Reply:
Just to check for configuration errors in the yml file, such as configuring the application to create 10^6 threads by mistake. I can remove the limit if you don't like it.

Reply:
So remove it please

Reply:
Done.